tests/database/odbc-base/Makefile

```diff
@@ -0,0 +1,42 @@
+include ../../common.mk
+
+PROG=odbc_app
+# PROG_SRC=$(PROG).c
+IMAGE_SIZE=50M
+
+EXECUTION_TIMEOUT=120
+SGXLKL_VERBOSE=0 
+SGXLKL_ENV=SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=1 SGXLKL_GETTIME_VDSO=0 SGXLKL_TAP=sgxlkl_tap0 SGXLKL_HOST_IMPORT_ENV="DB_SERVER_NAME,DB_NAME,DB_PASSWORD,DB_USERID"
```

Comment:
Split this into multiple lines with `\`.

Reply:
Sorry for the confusion, my personal preference is resolve one comment in local repo then mark resolve remotely. And after all problem resolved push it to remote.